static/css/custom.css

```diff
@@ -1389,3 +1389,8 @@ div.quire-caption-container > span code {
   color: #1a1a1a;
   line-height: 2;
 }
+
+
+#community-mcn-ku-event .quire-page__content .container .content { 
```

Comment:
Would be better to put this CSS inside the `<style>` tags in the MD file. That way we're not cluttering the main css file with things that are only applicable to temporary pages. I'd forgotten we'd done it that way last year.

Reply:
done!